content/appleboy/drone-line/index.md

```diff
@@ -168,73 +169,3 @@ locations
 
 stickers
 : a vaild sticker format
-
-# Template Reference
```

Comment:
If these are still valid, it should link somewhere for users to reference (like http://plugins.drone.io/drone-plugins/drone-slack/).

Reply:
Yes. we can merge this PR before document the template lib and I will update the `drone-line` library.